packages/manager/apps/dedicated/client/app/dedicatedCloud/terminate/dedicatedCloud-terminate.html

```diff
@@ -12,7 +12,28 @@
             <span data-translate="dedicatedCloud_close_service_warning"></span>
         </oui-message>
 
-        <p data-translate="dedicatedCloud_close_service_desc"></p>
-        <p data-translate="dedicatedCloud_close_service_question"></p>
+        <form
+            data-ng-if="$ctrl.canDeleteAtExpiration"
+            class="mt-4"
+            name="cancelSubscriptionForm"
+            novalidate
+        >
+            <oui-field
+                label="{{:: 'server_close_service_form_label' | translate }}"
```

Comment:
```suggestion
                data-label="{{:: 'server_close_service_form_label' | translate }}"
```

Reply:
As mentioned by @antleblanc https://github.com/ovh/manager/pull/3466#discussion_r472802183 We are shifting on this behavior. Only for webcomponents, meaning that on <oui-field>, we are ok to stop using them.